README.md

```diff
@@ -278,6 +278,14 @@ Net::HTTP.get('www.example.com',
   '/thing/5.json?x=1&y=2&z=3&anyParam=4')    # ===> Success
 ```
 
+### Allow query params
+
+```ruby
+stub_request(:get, "www.example.com").with(query: :any)
```

Comment:
Even this usage still feels a little heavy-weight to me.... Is there any reason we can't do `stub_request(:get, "www.example.com", query: :any)`? Or even just make it the default?